docs/transparent-validation/msi-z690/laboratory-assembly-guide.md

```diff
@@ -303,3 +303,8 @@ Example output:
 
 * `1` means that the platform is turned on.
 * `0` means that the platform is turned off.
+
+### USB devices
+
+Since some issues with USB controllers may only happen on select USB ports,
+it's important to plug in USB devices to all 17 USB ports of the board.
```

Comment:
For the MSI, we should avoid some ports due to this known issue: https://github.com/Dasharo/dasharo-issues/issues/437 :disappointed:

Reply:
Removed old contents and added info on faulty 3.x ports in commit https://github.com/Dasharo/docs/pull/816/commits/2d6001c0fec6c3cb8f896b372fd985fee278dac6